components/odh-notebook-controller/controllers/notebook_runtime_test.go

```
@@ -34,107 +35,196 @@
+						g.Expect(err).ToNot(HaveOccurred())
+					}, timeout, interval).Should(Succeed())
+
+					if testCase.ConfigMap == nil {
+						// Let's check the content of the ConfigMap created from the given ImageStream.
+						// TODO
+						Expect(configMap.GetName()).To(Equal(RuntimeImagesCMName))
+						expectedData := map[string]string{
+							"python-3.11-ubi9.json": `{"display_name":"Python 3.11 (UBI9)","metadata":{"display_name":"Python 3.11 (UBI9)","image_name":"quay.io/modh/odh-pipeline-runtime-datascience-cpu-py311-ubi9@sha256:5aa8868be00f304084ce6632586c757bc56b28300779495d14b08bcfbcd3357f","pull_policy":"IfNotPresent","tags":["some-tag"]},"schema_name":"runtime-image"}`,
+						}
+						Expect(configMap.Data).To(Equal(expectedData))
+					} else {
```

golangci-lint (1.64.8)

SA9003: empty branch

(staticcheck)